flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecGlobalGroupAggregate.java

```diff
@@ -78,6 +78,13 @@
 @ExecNodeMetadata(
         name = "stream-exec-global-group-aggregate",
         version = 1,
+        consumedOptions = {
+            "table.optimizer.simplify-operator-name-enabled",
+            "table.local-time-zone",
```

Comment:
this we can skip, we can assume that every operator needs this information esp when dealing with expressions. how do we use the time zone in this node?